src/contants/routes.ts

```diff
@@ -0,0 +1,13 @@
+export const paths = {
+  login: "/login",
+  myPatches: "/my-patches",
+  task: "/task",
+  patch: "/patch"
+};
+
+export const routes = {
+  login: paths.login,
+  myPatches: paths.myPatches,
+  task: `${paths.task}/:taskID/:tab?`,
+  patch: `${paths.patch}/:patchID`
+};
```

Comment:
I made routes into constants because we will be referencing these routes throughout the app. Using constants prevents accidentally changing a route and causing a cascade of errors from having the route hardcoded in other places.